doc/user/sql-grammar/sql-grammar.bnf

```diff
@@ -321,6 +321,7 @@ sink_kafka_connector ::=
     'KAFKA BROKER' host 'TOPIC' topic-prefix
     ('KEY' '(' key_column ( ',' key_column )* ')')?
     ('WITH' '(' ( field '=' val ) ( ( ',' field '=' val ) )* ')')?
+    ('CONSISTENCY' 'TOPIC' topic ('CONSISTENCY' 'FORMAT' 'AVRO USING' 'CONFLUENT SCHEMA REGISTRY' url)?)?
```

Comment:
I think this wants to be one line higher?

Reply:
Yes! Thank you -- diagram in the PR description has been updated as well.